dev-python/shiboken/shiboken-9999.ebuild

```diff
@@ -1,38 +1,41 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
 
 inherit cmake-utils python-r1 git-r3
 
 DESCRIPTION="A tool for creating Python bindings for C++ libraries"
-HOMEPAGE="https://wiki.qt.io/Pyside"
+HOMEPAGE="https://wiki.qt.io/PySide2"
 EGIT_REPO_URI=(
-	"git://code.qt.io/pyside/${PN}.git"
-	"https://code.qt.io/git/pyside/${PN}.git"
+	"git://code.qt.io/pyside/shiboken.git"
+	"https://code.qt.io/git/pyside/shiboken.git"
 )
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS=""
 IUSE="test"
-
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+# Minimum Qt version required, While PySide2 requires specific Qt 5 releases,
+# Shiboken2 purports to support all Qt 5 releases.
+QT_PV="5:5="
```

Comment:
> If shiboken only uses/wraps the public portion of the Qt API...

I'm fairly certain that shiboken necessarily inspects both the public and private portions of the Qt API. For example, [this unresolved shiboken issue](https://bugreports.qt.io/browse/PYSIDE-434) suggests that:

> "This class [QScreen] was not implemented for a good reason: The shiboken parser is not able to understand the construction. The problem is the **initialisation of private members, which shiboken interprets wrong.**

I might just be misinterpreting that statement, but shiboken *does* seem to be breaking privacy encapsulation... probably because it's got to.

> Qt maintains backward API & ABI compatibility across minor versions

I don't quite know how to interpret this, but the [Qt ABI Tracker](https://abi-laboratory.pro/tracker/timeline/qt) *does* show a variety of symbol changes across even minor versions. Moreover, minor versions appear to occasionally introduce entirely new components. Qt 5.5.1, for example, added the new [`Qt5WaylandClient` component](https://abi-laboratory.pro/tracker/objects_report/qt/5.5.0/5.5.1/report.html).

Even if minor versions *did* perfectly preserve backward compatibility, creating Python bindings for new components added between minor versions would still require recompiling shiboken on Qt version bumps.

I think, anyway? Admittedly, I'm in over my head here. And I'm not a tall guy to begin with.